Update documentation for mds to include plugin mds support (#7076)
opensearch-project/documentation-website: _dashboards/management/multi-data-sources.md

    - After you have entered the appropriate details in all of the required fields, the **Test connection** and **Create data source** buttons become active. You can select **Test connection** to confirm that the connection is valid.

> 3. Select **Create data source** to save your settings. The connection is created. The active window returns to the **Data Sources** main page, and the new connection appears in the list of data sources. The first create data source is marked as your default data source. 

Reviewer: ```suggestion
3. Select **Create data source** to save your settings. The connection is created, and the new data source appears in the list on the **Data Sources** main page. The first data source you create is marked as your default. 
```

4. Edit or update a data source connection.

>     - To make changes to the data source connection or mark a different data source as default, select a connection in the list on the **Data Sources** main page. The **Connection Details** window opens.

Reviewer: ```suggestion
    - On the **Data Sources** main page, select the connection you want to modify. The **Connection Details** window opens.
```

>     - To mark the current data source as default, 

Reviewer: ```suggestion
    - To mark the selected data source as the default, select the **Set as default** option. 
```

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/multidata-hide-localcluster.gif" alt="Multiple data sources hide local cluster">{: .img-fluid}

## Use multiple data sources in external dashboards plugins

> Starting 2.14, multiple data source is supported in the following plugins.

Reviewer: ```suggestion
The following plugins now support multiple data sources:
```

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/multidata-hide-localcluster.gif" alt="Multiple data sources hide local cluster">{: .img-fluid}

## Using multiple data sources with external dashboards plugins
Introduced 2.14
{: .label .label-purple }

The following plugins now support multiple data sources

### Index management

> When the data source feature is enabled, you can navigate to **Index Management** under the **Management** menu. Using indexes as an example, you can view all connected data sources and select a specific one from the navigation bar at the upper right. By default, the indexes from the designated default data source is selected are displayed. However, you can select any connected data source to view its corresponding indexes. The following GIF illustrates these steps.

Reviewer 2: ```suggestion
When the data source feature is enabled, you can navigate to **Index Management** under the **Management** menu. Using indexes as an example, you can view all connected data sources and select a specific one from the navigation bar on the upper right. By default, the indexes from the designated default data source are displayed. However, you can select any connected data source to view its corresponding indexes. The following GIF illustrates these steps.
```

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ism_mds1.gif" alt="Multiple data sources in ISM list page"/>

> To perform operations on a specific index within a data source, select the individual index from the list. To create a new index, select the `Create Index` button, which opens a form. Fill in the required information and select the `Create` button. The index is created within the selected data source. The following GIF illustrates these steps. 

Reviewer 2: Should the names of the buttons be bold instead of in code font?

Author: sure, change name of button to bold in new commit

Reviewer: @natebower @BionIT I thought I changed them all. I'll format.

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ism_mds2.gif" alt="Multiple data sources in ISM create page"/>

### Anomaly detection

> When the data source feature is enabled, you can navigate to **Anomaly Detection** under the **OpenSearch Plugins** menu. In the navigation bar at the upper right, you can view all connected data sources and select a specific data source to view the dashboard from that source if the data source has detectors. If the selected data source does not have any detectors, the page prompts you to `Create detector`. The following GIF illustrates these steps.

Reviewer 2: ```suggestion
When the data source feature is enabled, you can navigate to **Anomaly Detection** under the **OpenSearch Plugins** menu. On the navigation bar on the upper right, you can view all connected data sources and select a specific data source to view the dashboard from that source if it has detectors. If the selected data source does not have any detectors, the page prompts you to `Create detector`. The following GIF illustrates these steps.
```

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ad_mds1.gif" alt="Multiple data sources in Anomaly Detection dashboard page"/>

> When you select **Detectors** from the side bar, the page displays the detectors currently configured for the currently selected data source. You can view and configure individual detectors by selecting them from the list. These steps are shown in the following GIF.

Reviewer 2: ```suggestion
When you select **Detectors** from the side bar, the page displays the detectors currently configured for the selected data source. You can view and configure individual detectors by selecting them from the list. These steps are shown in the following GIF.
```

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ad_mds2.gif" alt="Multiple data sources in Anomaly Detection detector page"/>

### Security

> When the data source feature is enabled, you can navigate to **Security** under the **Management** menu. Using role management as an example, you can view all connected data sources in the navigation bar on the upper right and select a specific data source to view its existing roles. To create a new role, select the **Create role** button, which takes to a new create page. Enter the required information and select **Create** to add the new role to the selected data source. The following GIF illustrates these steps.

Reviewer 2: ```suggestion
When the data source feature is enabled, you can navigate to **Security** under the **Management** menu. Using role management as an example, you can view all connected data sources in the navigation bar on the upper right and select a specific data source to view its existing roles. To create a new role, select the **Create role** button, which takes you to a new page. Enter the required information and select **Create** to add the new role to the selected data source. The following GIF illustrates these steps.
```

> When the data source feature is enabled, you can navigate to **Security** under the **Management** menu. Using roles management as an example, you can view all connected data sources in the navigation bar at the upper right and select a specific data source to view its existing roles. To create a new role, select the **Create role** button, which takes to a new create page. Fill in the required information and select **Create** to add the new role to the selected data source. The following GIF illustrates these steps.

Reviewer 2: ```suggestion
When the data source feature is enabled, you can navigate to **Security** under the **Management** menu. Using role management as an example, you can view all connected data sources in the navigation bar on the upper right and select a specific data source to view its existing roles. To create a new role, select the **Create role** button, which takes to a new create page. Enter the required information and select **Create** to add the new role to the selected data source. The following GIF illustrates these steps.
```

> When the data source feature is enabled, you can navigate to **Security** under the **Management** menu. Using roles management as an example, you can view all connected data sources in the navigation bar at the upper right and select a specific data source to view its existing roles. To create a new role, select the **Create role** button, which takes to a new create page. Fill in the required information and select **Create** to add the new role to the selected data source. The following GIF illustrates these steps.

Reviewer 2: The end of the third sentence needs some correction.

Author: Could you clarify?

Reviewer 2: I suspect instead of "which takes to a new create page", we mean "which takes you to a new page."

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/security_mds1.gif" alt="Multiple data sources in Security plugin"/>

### Maps

> When the data source feature is enabled, you can navigate to **Maps** under the **OpenSearch Plugins** menu. To edit an existing map, select it from the maps list page, which opens the edit page. On the edit page, you can view all available data sources and the ones currently used in the map. To add a new layer, select **Add layer**, then select **Documents** from the prompt, which opens a flyout. In the flyout, select the index pattern and geospatial field. Note that the data source name is prefixed to the index pattern name. After selecting **Update**, the new layer is added. Select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/database-icon.png" class="inline-icon" alt="database icon"/>{:/} icon to verify that a new data source is now being used in the map. The following GIF illustrates these steps.

Reviewer 2: ```suggestion
When the data source feature is enabled, you can navigate to **Maps** under the **OpenSearch Plugins** menu. To edit an existing map, select it from the maps list page, which opens the edit page. On the edit page, you can view all available data sources and the ones currently used in the map. To add a new layer, select **Add layer**, and then select **Documents** from the prompt, which opens a flyout. In the flyout, select the index pattern and geospatial field. Note that the data source name is prefixed to the index pattern name. After selecting **Update**, the new layer is added. Select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/database-icon.png" class="inline-icon" alt="database icon"/>{:/} icon to verify that a new data source is now being used in the map. The following GIF illustrates these steps.
```

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ml_mds1.gif" alt="Multiple data sources in Machine Learning Plugin"/>

### Notifications

When the data source feature is enabled, you can navigate to **Notifications** under the **Management** menu. The page displays the notification channels configured for the currently selected data source. To view channels from a different data source, select the desired data source from the menu. To view or edit the details of an existing channel, select it from the list, which opens the channel details page. The following GIF illustrates these steps.

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/notification_mds1.gif" alt="Multiple data sources in Notification plugin"/>

### Search relevance

> When data source feature is enabled, you can navigate to **Search Relevance** under the **OpenSearch Plugins** menu. In the navigation bar at the upper right, you can view all available data sources. To compare search results between indexes from different data sources, first select a data source and an index for **Query 1**, then select a data source and an index for **Query 2**. Select **Search** to run the queries. The following GIF illustrates these steps. 

Reviewer 2: ```suggestion
When the data source feature is enabled, you can navigate to **Search Relevance** under the **OpenSearch Plugins** menu. On the navigation bar on the upper right, you can view all available data sources. To compare search results between indexes from different data sources, first select a data source and an index for **Query 1**, and then select a data source and an index for **Query 2**. Select **Search** to run the queries. The following GIF illustrates these steps. 
```

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/searchrelevance_mds1.gif" alt="Multiple data sources in Search Relevance plugin"/>

## Next steps 

> After configuring multiple data sources, you can begin exploring the data from each source. Refer to the following resources for more information:

Reviewer 2: ```suggestion
After configuring multiple data sources, you can analyze the data from each source. Refer to the following resources for more information:
```
